lexorus/flickr-search
# Feature/GitHub actions

### Background
Hope I don't need to explain what are the benefits of CI 🙃. At the moment CI workflow will do the following:
1. Checkout
2. Try to restore cocoapods cache
3. Install cocoapods dependencies
4. Build and test

### What has been done?
1. Add CI through Github Actions
2. Remove override of ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES in test target

### How to test?
1. Build and test should pass
2. When running the checks the second time, the cocoapods cache should be restored and `Install dependencies` step should take less time.
